### ISSUE TITLE
Fingerprint unlock

### ISSUE THREAD

**Issue by federicoiosue** [federicoiosue/Omni-Notes#275]

**Comment by seancojr** [comment]
The more I have become accustomed to securing many of the apps I have installed on my phone, it pleases me to learn when one of them  supports fingerprint authentication. This planned feature will be greatly appreciated.

**Comment by federicoiosue** [comment]
@seancojr yes it is! I'm a great fan of that feature since I'm using a device capable of it so I'll implement that for sure into Omni Notes

**Comment by DanielKaparunakis** [comment]
I would like to work on this feature.

**Comment by federicoiosue** [comment]
That's actually targeted for pro version of the app @DanielKaparunakis so I discourage you from working on that due to the fact that'll not be released as a free feature!

**Comment by DanielKaparunakis** [comment]
Is the pro version open sourced?

**Comment by federicoiosue** [comment]
@DanielKaparunakis I'd really like to. 

I think to try step-by-step in that direction eventually by starting developing without open-sourcing the code initially.
I never developed open-source paid software so I've to admit that I'm missing experience on its related commercial approaches...
